codegen/tests/simple/types.go

```diff
@@ -23,6 +25,39 @@ type Item struct {
 	Name string `json:"-"`
 }
 
+// PostRequest ...
+type PostRequest struct {
+	Bt *bool            `json:"Bt,omitempty"`
+	Dt convert.JSONTime `json:"Dt"`
+	St string           `json:"St"`
+}
+
+func (t *PostRequest) UnmarshalJSON(data []byte) error {
+	inner := struct {
+		Bt *bool
```

Comment:
I suspect these require struct tags for json also:
```go
Bt *bool            `json:"Bt,omitempty"`
```

Reply:
ok